Automattic/wp-calypso
# Plans: Make single product icons darker blue

### Changes proposed in this Pull Request

This PR makes single product icons darker blue (#2271B1).

Single products include:
1. Jetpack Backup
2. Jetpack Search
3. Jetpack CRM
4. Jetpack Scan
5. Jetpack Anti-spam

### Testing instructions

- Run this PR locally (both Calypso and Jetpack Cloud)
- Go to the new Plans grid page (`/plans/:site`).
- Turn **off** the feature flag (append `?flags=-plans/alternate-selector` to the url.) and verify 5 products above are showing the original lighter blue icons (See "Before" images below).
- Next, turn **on** the feature flag (remove the `?flags=-plans/alternate-selector` from the url.) and verify the 5 products above are now showing the new darker blue icons (See "After" images below). (#2271B1)
- Verify the darker blue icons are showing in Calypso and Jetpack Cloud.

**BEFORE**

![Screenshot on 2020-10-12 at 06-38-13](https://user-images.githubusercontent.com/11078128/95755652-86e96f00-0c59-11eb-96b0-87017bc26971.png)
![Screenshot on 2020-10-12 at 06-38-58](https://user-images.githubusercontent.com/11078128/95755665-89e45f80-0c59-11eb-912c-fae474fb356c.png)
![Screenshot on 2020-10-12 at 06-39-35](https://user-images.githubusercontent.com/11078128/95755676-8cdf5000-0c59-11eb-8949-efd7e6852e75.png)
![Screenshot on 2020-10-12 at 06-39-58](https://user-images.githubusercontent.com/11078128/95755686-8fda4080-0c59-11eb-9dde-aaa34b40ab39.png)
![Screenshot on 2020-10-12 at 06-40-26](https://user-images.githubusercontent.com/11078128/95755692-923c9a80-0c59-11eb-99ef-ad2a65d935e2.png)

**AFTER**  (They should look like these:)

![Screenshot on 2020-10-12 at 06-41-37](https://user-images.githubusercontent.com/11078128/95755790-abdde200-0c59-11eb-84eb-22eefb92c4c8.png)
![Screenshot on 2020-10-12 at 06-42-10](https://user-images.githubusercontent.com/11078128/95755802-b009ff80-0c59-11eb-80bd-0c79f5fbe415.png)
![Screenshot on 2020-10-12 at 06-42-39](https://user-images.githubusercontent.com/11078128/95755806-b1d3c300-0c59-11eb-8611-a17bd82022ee.png)
![Screenshot on 2020-10-12 at 06-43-02](https://user-images.githubusercontent.com/11078128/95755816-b5674a00-0c59-11eb-8f2d-a41517ce0711.png)
![Screenshot on 2020-10-12 at 06-43-23](https://user-images.githubusercontent.com/11078128/95755827-b8623a80-0c59-11eb-991e-c04700e3f755.png)


Fixes: 1196341175636977-as-1198175394063569/f
